### ISSUE TITLE
enable arm64 testing

### ISSUE THREAD

**Issue by gdams** [AdoptOpenJDK/openjdk-tests#81]
I am currently sorting out a second arm64 machine for testing our arm builds... Can we start the setup process for arm64 machines?

**Comment by sophia-guo** [comment]
What's the machine label? Is the machine ready with all related tools installed?

**Comment by gdams** [comment]
I will be provisioning a machine in the next week or so

**Comment by gdams** [comment]
test-packet-armv8-ubuntu-16.04 should now be fully setup

**Comment by gdams** [comment]
@AdoptOpenJDK/openjdk-tests have we made any progress on this yet?

**Comment by sophia-guo** [comment]
I was on vacation last week. Will try it this week. Assume all related tools installed( example wget, unzip,...)

**Comment by gdams** [comment]
@sophia-guo all of the tools should be on the machine

**Comment by sophia-guo** [comment]
@gdams job using label "arm64&&test" is assigned to build-linaro-armv8-centos7-2, which should be a build machine?

Suppose job should be running on test-packet-armv8-ubuntu-16.04, which is not in the list of 
https://ci.adoptopenjdk.net/label/test/

**Comment by sophia-guo** [comment]
https://github.com/AdoptOpenJDK/openjdk-tests/pull/85 is to enable tests on arm. Seemed arm build is deleted? Any plan to re-enable it? @gdams , Thanks.

**Comment by sophia-guo** [comment]
Build #88 under test_x86-64_experimental is a test build against arm64, which takes 5 hours and 782 failures( 210 are hotspot tests) . https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_x86-64_experimental/88/#showFailuresLink

@smlambert @gdams we need a standardize Jenkins Label Schema same as openj9. For now @gdams could you please add "x64" label to this arm64 test machine?

@gdams the other question is I didn't see any openjdk8_build_arm64_linux build. Did see openjdk8_build_arm32_linux build. In https://api.adoptopenjdk.net/openjdk8/releases/aarch64_linux only arm64 is available. Just to confirm we only need to test arm64 sdk? Thanks.

**Comment by sophia-guo** [comment]
@gdams is https://ci.adoptopenjdk.net/computer/test-packet-armv8-ubuntu-16.04/ good to go? Or extra machine will be added?

**Comment by gdams** [comment]
it should be good

**Comment by sophia-guo** [comment]
Checked all arm64 builds. For openjdk8 we only have release version. For openjdk9 we have  nightly build https://ci.adoptopenjdk.net/job/openjdk9_build_arm64_linux/. 

Just to confirm do we only need test build for openjdk arm64? Or we also need test build for openjdk8 arm64, which can be triggered manually.

**Comment by karianna** [comment]
@sophia-guo Ideally we'd have a test for both `openjdk9 arm64` and `openjdk8 arm64`.

I think the test build for arm32 is desirable but a nice to have right now (perhaps create a separate issue).

**Comment by sophia-guo** [comment]
what's the IP address of test-packet-armv8-ubuntu-16.04? @gdams I believe there is a repo or website to check each machine's IP, I just can't find it.

**Comment by gdams** [comment]
https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/infrastructure.md

**Comment by smlambert** [comment]
@sophia-guo 
test-packet-armv8-ubuntu-16-04  =>  147.75.74.50 

It will be good to enable the tests we can, report what the remaining test builds are blocked by and then close this item.  We will address the parallel test story in a separate issue so that more tests can potentially be run (faster) if/when more than 1 test machine becomes available.

--------------------------------------

Enabled openjdk test build: (currently restricted to running jdk_math group)
https://ci.adoptopenjdk.net/view/OpenJDK%20tests/job/openjdk9_hs_openjdktest_aarch64_linux/ 

Not running: (old style test trigger, should be updated to use JenkinsfileBase)
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk9_test_arm64_linux/  


---------------------------------------

Enabled system tests:
https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk9_hs_systemtest_aarch64_linux/

Disabled system test builds, @Mesbah-Alam, can these be enabled? there appears to be a regular build running :
https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk10_hs_systemtest_aarch64_linux/

Disabled awaiting builds:
https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk9_j9_systemtest_aarch64_linux/ -
triggered by https://ci.adoptopenjdk.net/job/openjdk9_openj9_build_arm64_linux/

https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk8_hs_systemtest_aarch64_linux/ - to be triggered by openjdk8_build_arm64_linux

https://ci.adoptopenjdk.net/view/System%20tests/job/openjdk8_j9_systemtest_aarch64_linux/ - to be triggered by openjdk8_openj9_build_arm64_linux

**Comment by sophia-guo** [comment]
Latest openjdk test build on arm with only jdk_math group enabled passed.
https://ci.adoptopenjdk.net/view/OpenJDK%20tests/job/openjdk9_hs_openjdktest_aarch64_linux/36/

@ShelleyLambert the old style one doesn't help much probably delete it. Thanks.

**Comment by smlambert** [comment]
👍thanks @sophia-guo - do you mean I can delete this one openjdk9_test_arm64_linux ?

**Comment by sophia-guo** [comment]
Yes, please. Thanks!

**Comment by smlambert** [comment]
Closing this as we have enabled as much testing as there are builds to trigger them.
